.cz-config.js

```diff
@@ -1,21 +1,25 @@
-const { execSync } = require('child_process');
+const fs = require('fs')
```

Comment:
This change makes me so happy btw – the slight delay every time I committed made a little part of me die each time 🤣